### PR TITLE
T13 patch

### PR DESCRIPTION
**Pull request by danielguedesb**
- Add t13 in callout-vendor.js
- Edit rtc-documentation and rtc-implementation

### CODE REVIEW

File: extensions/amp-a4a/rtc-documentation.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ The `errorReportingUrl` property is optional. The only available macros are ERRO
 - Yieldbot
 - Kargo
 - Yieldlab
+- T13
```
Comment (powerivq) [inline extensions/amp-a4a/rtc-documentation.md:144]: Nit: I know there are out-of-place vendors here, but could you help putting it in alphabetical order?

File: extensions/amp-a4a/rtc-publisher-implementation-guide.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ To use RTC, you must meet the following requirements:
 - Yieldbot
 - Kargo
 - Yieldlab
+- T13
```
Comment (powerivq) [inline extensions/amp-a4a/rtc-publisher-implementation-guide.md:39]: Nit: order

File: extensions/amp-a4a/rtc-documentation.md (hunk 1 above)
Comment (danielguedesb) [reply extensions/amp-a4a/rtc-documentation.md:144]: Gladly, cheers

**Comment by powerivq** [comment]
Mind refactoring to rerun unreported percy test

**Pull request merged**